src/lib/Libwin/accesinfo.c

```diff
@@ -84,6 +84,10 @@ create_secure_dacl(char *user, ACCESS_MASK mask, SID *owner_sid)
 
 	rids[0] = DOMAIN_ALIAS_RID_ADMINS;
 	k = getgids(getlogin(), grp, rids);
+	if (k <= 0) {
+		sprintf(logb, "failed to get group-id for %s user", getlogin());
+		log_err(-1, __func__, logb);
```

Comment:
errno?

Reply:
As this (getgids) is pbs function and we already handled error no login in corresponding steps inside that function depending on which api failed, so didn't log the code again here.

Reply:
why repeat getlogin() again just to get the username?

Reply:
yes